.github/workflows/dl-dev-tag-docker-push.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish Docker image
+on:
+  push:
+    tags:
+      - orbyter-dl-dev-[0-9].*
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Get version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/orbyter-dl-dev-}
```

Comment:
What is this witchcraft you're doing here?

Reply:
Where is GITHUB_REF defined? Is that some default environment variable?

Reply:
I got this from [here](https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions). `GITHUB_REF` is an env var (or at least you can think of it as one) that tells your the reference tag of the commit--e.g., `git tag mikesfuntag` corresponds to `GITHUB_REF=refs/tags/mikesfuntag`. The `#refs/tags/...` strips the `refs/tags/...` from the string.